src/kemal/exception_handler.cr

```diff
@@ -11,12 +11,44 @@ module Kemal
     rescue ex : Kemal::Exceptions::CustomException
       call_exception_with_status_code(context, ex, context.response.status_code)
     rescue ex : Exception
+      # Use error handler defined for the current exception if it exists
+      return call_exception_with_exception(context, ex, 500) if Kemal.config.error_handlers.has_key?(ex.class)
```

Comment:
**suggestion:** This special case with `has_key?` seems unnecessary because this is already covered with `ex.class <= key`.
It should be safe to remove this partial duplication.
```suggestion
```